src/Server/Log.idr

```diff
@@ -3,55 +3,18 @@
 ||| (C) The Idris Community, 2021
 module Server.Log
 
-import Core.Core
 import Core.Context.Context
+import Core.Core
 import Core.Directory
 import Server.Configuration
+import Server.Severity
 import Server.Utils
 import System.Directory
 import System.File
 import System.Path
 
 %default total
 
-||| Type for the severity of logging messages.
```

Comment:
Separating this into its file is needed, as a circular dependency was introduced. The configuration now depends on the Severity, and logging depends on the configuration.